src/types.ts

```diff
@@ -336,21 +336,30 @@ export type NftTokens = {
   [nftTokenId: string]: NftToken,
 };
 
-export type NftCollections = {
-  [nftCollectionsAddress: string]: NftTokens,
-};
-
-export type NftContract = {
+export type NftContractInfo = {
   chain: string,
   name: string,
   symbol: string,
   contractAddress: string,
+  deployAddress: string,
+  totalSupply: number,
+  lastBlockNumber: number,
+};
+
+export type NftContract = {
```

Comment:
Is this change backward compatible?

Reply:
Should we need backward compatibility after upgrading the version? I thought this typing can notify that should need to change about this.

Reply:
I reverted backward compatibility!